os/arch/arm/src/amebasmart/amebasmart_timerisr.c

```diff
@@ -94,6 +91,13 @@ int up_timerisr(int irq, uint32_t *regs)
 	  delta_ticks = 1;
 #endif
 
+    u32 ticks_to_process = delta_ticks;
```

Comment:
this PR change should be applied regardless PM. right?

Reply:
The delta_ticks is updated in only CONFIG_PM
If CONFIG_PM is disabled, `delta_ticks = 1`. is it okay?

Reply:
Okay, sometime we are test non-CONFIG_PM case.
Please support non-CONFIG_PM case also, you can raise new PR after merge this